### ISSUE TITLE
'1p <any search>' from Alfred fails after setup.

### ISSUE THREAD

**Issue by adamreiswig** [alfredapp/1password-workflow#87]
### Frequently Asked Questions

- [X] I have [read the FAQ](../blob/main/FAQ.md) before opening the issue.

### Workflow version

2024.3

### Alfred version

5.5.1

### macOS version

15.1.1

### Debugger output

```alfred_debugger
### Workflow version
2024.3

### Alfred version
5.5.1

### macOS version
15.1.1

### Architecture
arm64

### Preferences
Default

### 1Password version
8.10.54

### op version
2.30.3

### Launchd job
NOT loaded
NOT installed
```

### More details

To the best of my knowledge I have completed the setup and nothing happens when I enter '1p <search>' in Alfred. This is a brand new Mac. 1Password is installed and the developer CLI option is checked. I have installed '1password-cli' from brew.  I have the latest 1Password workflow installed in Alfred. Independently, 1Password and Alfred are functioning as expected. From Alfred, I have ran extras force update. Nothing happens. When I run 'op item list' from the terminal, I get the biometric prompt and then a full list of my 1Password vault.

So, the issue is, when I type '1p' I get the option for 'sign in and update items'. When I do this, nothing happens. If I type '1p' and then a search item, space separated, it thinks it's a web search and takes me to google. What else can I check? Thanks.

**Comment by vitorgalvao** [comment]
You wrote three times that “nothing happens”. Please read [the FAQ](https://github.com/alfredapp/1password-workflow/blob/main/FAQ.md), don’t just tick the checkbox. I am unable to help if you do not follow the steps. It tells you exactly what to do. Provide debugger output, check for the Automation Tasks.

**Comment by adamreiswig** [comment]
My apologies. I did mention *2* separate specific actions where "nothing happens". I was tired and I'm sure I missed a couple things from the FAQ.

That said, I found the issue, even though I had run !1pdiagnose and :1pextras -> force update, the "automation tasks" were all empty. There was a "load" options button which I clicked and it loaded all of them, and now everything works. I'm guessing this is an Alred issue but I don't know.
